modules/ec2/security_group.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_security_group" "remote_security_group" {
+  name        = local.security_group_name
+  vpc_id      = var.instance_vpc_id
+  description = "Jump Server Security Group"
```

Comment:
Remove any references about "Jump Server".
In this case, the description could be "Remote Access Security Group"

Reply:
Done.-